common/src/main/java/gov/cms/ab2d/common/repository/JobRepository.java

```diff
@@ -38,4 +40,9 @@
     @Query("UPDATE Job j SET j.status = 'SUBMITTED' WHERE j.jobUuid IN :jobUuids ")
     void resetJobsToSubmittedStatus(List<String> jobUuids);
 
+
+    @Modifying
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    @Query(nativeQuery = true, value = "UPDATE job SET progress = :percentageCompleted WHERE job_uuid = :jobUuid ")
```

Comment:
Curious why we're having to use a native, non HQL, query here. Seems to be a simple query.

Reply:
Changed to HQL.